src/identity.ts

```diff
@@ -11,6 +11,7 @@ import { AddressInterface } from './types';
 export enum IdentityType {
   PrivateKey = 1,
   Mnemonic,
+  MasterPublicKey,
```

Comment:
As there is different behavior for the same interface (able-to-sign identity and readonly-one) what do you think about add a new method in `IdentityInterface` to check this. Something like `canSign(): boolean`

Reply:
I agree

Reply:
Can you take this?